crates/binstalk-types/src/cargo_toml_binstall/package_formats.rs

```diff
@@ -52,7 +52,7 @@ impl PkgFmt {
     pub fn extensions(self, is_windows: bool) -> &'static [&'static str] {
         match self {
             PkgFmt::Tar => &[".tar"],
-            PkgFmt::Tbz2 => &[".tbz2", ".tar.bz2"],
+            PkgFmt::Tbz2 => &[".tbz2", ".tar.bz2", ".tbz"],
```

Comment:
Is `.tar.bz` a valid extension for tbz2?

If it is, then I think we should also accept it:

```suggestion
            PkgFmt::Tbz2 => &[".tbz2", ".tar.bz2", ".tbz", ".tar.bz"],
```